### DISCUSSION TITLE
jdownloader retronas

### DISCUSSION THREAD

**Discussion by rogersta99** [retronas/discussions#77] [Q&A]
Is it possible to use jdwonloader with retronas?  If so, how?

**Comment by rumun1990** [comment]
I think, you can install jdownloader headless version on rpi, and have also retronas. I will set up in few days clean system with retronas and jdownloader. I hope it will work ;)

**Comment by sairuk** [comment]
I've added this to the ideas page. I'd forgotten about this post.

And yes you should just able to do a headerless installation, its been many years since i looked at jdownloader so you'd be best to check their forums on a linux headerless installation 

then ssh into your RetroNAS and follow their guides.

**Comment by rumun1990** [comment]
@rogersta99 if you want to try by yourself, you can type on web search "how to install jdownloader raspberry pi" and there will be instruction on blogspot,
